pumps/sql_aggregate.go

```diff
@@ -152,7 +150,15 @@ func (c *SQLAggregatePump) WriteData(ctx context.Context, data []interface{}) er
 			table = analytics.AggregateSQLTable
 		}
 
-		analyticsPerOrg := analytics.AggregateData(data[startIndex:endIndex], c.SQLConf.TrackAllPaths, c.SQLConf.IgnoreTagPrefixList, c.SQLConf.StoreAnalyticsPerMinute, false)
+		// if StoreAnalyticsPerMinute is set to true, we will create new documents with records every 1 minute
+		var aggregationTime int
+		if c.SQLConf.StoreAnalyticsPerMinute {
+			aggregationTime = 1
+		} else {
+			aggregationTime = 60
+		}
+
+		analyticsPerOrg := analytics.AggregateData(data[startIndex:endIndex], c.SQLConf.TrackAllPaths, c.SQLConf.IgnoreTagPrefixList, "", aggregationTime, false)
```

Comment:
@mativm02 why the dbIdentifier is empty here?

Reply:
@tbuchaillot because dbIdentifier is being used as the key of the map `lastDocumentTimestamp` we were talking before. This map is only being used for mongo. If the string is empty, it means the database != mongo (it's SQL in this case).
We can think about a cleaner way to handle this. Maybe have different functions to set timestamp to new data added to documents for every database service? Instead of handling all of them in one function. After setting the timestamp, we can continue using `AggregateData`.

Reply:
@mativm02 What would happen if you have 2 SQL Aggregate pumps running and writing data to different dbs?

Reply:
I will test that use case, but nothing should happen, SQL Aggregate should work as always 🤔